Makefile

```diff
@@ -0,0 +1,20 @@
+IMAGE=theyosh/terrariumpi
+VERSION=4.1.0
```

Comment:
I'm not sure if this is where you want to store your version numbers for docker containers, but this was handy for me.

Reply:
I will think I can fix that once this PR is merged